addons/l10n_id_efaktur/views/account_move_views.xml

```diff
@@ -64,9 +52,9 @@
             <field name="arch" type="xml">
                 <field name="name" position="after">
                     <field name="l10n_id_tax_number"/>
+                    <field name="l10n_id_attachment_id"/>
                     <group>
-                        <filter string="Tax CSV Not Uploaded" name="upload" domain="[('l10n_id_date_validated', '=', False), ('l10n_id_tax_number', '!=', False)]" />
-                        <filter string="Tax CSV Downloaded" name="is_downloaded" domain="[('l10n_id_attachment_id', '!=', False), ('l10n_id_date_validated', '=', False)]"/>
+                        <filter string="Tax CSV Not Uploaded" name="upload" domain="[ ('l10n_id_tax_number', '!=', False), ('l10n_id_attachment_id', '=', False)]" />
```

Comment:
Maybe we could use "To exchange in Tax CSV"   Should nto we filter out the cancelled/draft here as well?  (or is it filtered somewhere else?)